cogboard-app/src/main/kotlin/com/cognifide/cogboard/config/CredentialsConfig.kt

```diff
@@ -1,20 +1,6 @@
 package com.cognifide.cogboard.config
 
-import com.cognifide.cogboard.config.validation.credentials.CredentialsValidator
-import io.vertx.core.json.JsonArray
-import io.vertx.core.json.JsonObject
-
-class CredentialsConfig : Config() {
-
-    fun getCredentials(): JsonArray = load().getJsonArray(CREDENTIALS_ARRAY)
-
-    override fun validate(configJson: JsonObject): Boolean {
-        return CredentialsValidator.validate(configJson)
-    }
-
-    override fun filePath(): String = CREDENTIALS_CONFIG_FILE_PATH
-
-    override fun type(): ConfigType = ConfigType.CREDENTIALS
+class CredentialsConfig {
```

Comment:
`CredentialsConstants`? It doesn't configure anything now, just a bunch of constants ;)